### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by JustinKuli**
All directory names are now all lowercase, no hyphens or underscores.

Helm charts are now much more similar to each other, and have (generally) better/more consistent security settings.

Some other cleanup stuff.

### CODE REVIEW

**Comment by JustinKuli** [comment]
@ckandag I can't assign you as a reviewer (for some reason), but here is the PR for cleaning the repo up before transferring it to stolostron.

File: pkg/addon/configpolicy/manifests/managedclusterchart/values.yaml [hunk 1]
```diff
@@ -4,13 +4,10 @@ fullnameOverride: null
 nameOverride: null
 
 org: open-cluster-management
+replicas: 1
 
-enabled: true
 hubKubeconfigSecret: config-policy-controller-hub-kubeconfig
 
-clusterName: null
```
Comment (ckandag) [inline pkg/addon/configpolicy/manifests/managedclusterchart/values.yaml:11]: I know clustername is set through the addonframework but still  I would leave these in the values.yaml as null.. just so all variables in the chart are listed/published  in values.yaml and can be identified by just looking at the values.yaml

File: pkg/addon/iampolicy/agent_addon.go [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package config_policy
+package iampolicy
```
Comment (ckandag) [inline pkg/addon/iampolicy/agent_addon.go:1]: a bad copy/paste job by me. :(

File: pkg/addon/certpolicy/manifests/managedclusterchart/templates/deployment.yaml [hunk 3]
```diff
@@ -49,62 +42,51 @@ spec:
...
-          timeoutSeconds: 5
-        readinessProbe:
-          exec:
-            command: ["sh", "-c", "exec echo start certificate-policy-controller"]
-          initialDelaySeconds: 10
-          timeoutSeconds: 2
         env:
         - name: WATCH_NAMESPACE
-          value: ""
+          value: "{{ .Values.clusterName }}"
```
[the remark this answers is not in the document]
Comment (JustinKuli) [reply pkg/addon/certpolicy/manifests/managedclusterchart/templates/deployment.yaml:47]: Other controller just watch the cluster namespace for their policies - I thought they would all appear there from the template sync.

File: pkg/addon/configpolicy/manifests/managedclusterchart/values.yaml (hunk 1 above)
Comment (JustinKuli) [reply pkg/addon/configpolicy/manifests/managedclusterchart/values.yaml:11]: 👍 It's just moved later in this file, to match the other charts' values.yaml format.

**Review by ckandag — APPROVED** [review]

**Pull request merged**